deployment/helm-k8s/templates/response/deployment.yaml

```diff
@@ -48,15 +48,22 @@ spec:
           initialDelaySeconds: 20
           periodSeconds: 10
         env:
+        {{ if .Values.tls.enabled -}}
```

Comment:
meybe this si confusing since these certs are for a client to store trusted certificates